website/content/docs/install-boundary/initialize.mdx

```diff
@@ -46,14 +45,9 @@ kms "aead" {
 <truncated>
 ```
 
-In this example, we're using hardcoded AEAD keys, but in a real world non-dev deployment, you
-should use your cloud provider's KMS such as [AWS KMS](/boundary/docs/configuration/kms/awskms) to manage the keys Boundary
-uses to encrypt sensitive information.
-
-The KMS block we're focused on is the `recovery` block. This block specifies the key used to "recover" Boundary
-but you can also use it to authenticate to Boundary and manage it as a "global" super user. This allows
-you to authenticate from the CLI or from Terraform in order to manage Boundary without any generated
-resources.
+The KMS with the `recovery` purpose is used to "recovery" Boundary, allowing you to authenticate to Boundary and managed
+it as a super user. It also allows you to authenticate from the CLI or from Terraform in order to manage BOundary
+without any generated resources.
```

Comment:
```suggestion
The KMS with the `recovery` purpose is used to "recover" Boundary, allowing you to authenticate to Boundary and manage
it as a super user. It also allows you to authenticate from the CLI or from Terraform to manage Boundary
without any generated resources.
```

Reply:
I think you meant "recover" instead of "recovery" here, but please let me know if I misunderstood.